docs/effective-volto/about_effective_volto.md

```diff
@@ -0,0 +1,37 @@
+---
+html_meta:
+  "description": ""
+  "property=og:description": ""
+  "property=og:title": ""
+  "keywords": ""
```

Comment:
Please add MyST topmatter. Also make sure that `myst` is the root key wherever you create a new page and the remaining keys are indented appropriately.

```
myst:
  html_meta:
    "description": "About Effective Volto"
    "property=og:description": "About Effective Volto"
    "property=og:title": "About Effective Volto"
    "keywords": "Volto, Tiberiu Ichim, Víctor Fernández de Alba"
```